### ISSUE TITLE
Logbook import from LLP - converted .logbook file does not capture ICM/standalone/sessions

### ISSUE THREAD

**Issue by Pineapple1168** [anaestheticsapp/logbook-standard#3]
Hi,

I have just registered and tried to import my logbook as instructed from LLP. The converted .logbook file does not capture the non-anaesthetic cases - is that a current limitation or will I able to correct the issue?

**Comment by anaestheticsapp** [comment]
Yep, it's only possible to import anaesthetic cases. The rest will follow at some point. Shouldn't take long to implement but currently very busy so this will probably take a few weeks. Will posts updates to this here.
